docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
+<img src="images/EditRefrence1.png" width="600"/>
+<img src="images/EditRefrence2.png" width="600"/>
+<img src="images/EditRefrence3.png" width="600"/>
```

Comment:
<img width="305" alt="ss10" src="https://user-images.githubusercontent.com/60163283/140519851-d9fd1891-f7e0-44f5-bc70-5ea002e020c9.PNG">
Once u replace instrument with stock, list down all the editXYZ methods in editGeneralParameter (editName, editCurrentPrice, editSentiment, editDoneStatus), then afterwards selfinvocate to editRemark
Remove the box note and replace that with a note stating that other instrument specific parameters will call the relevant editXYZ method. Also write a note stating what XYZ stands for.